src/main/java/it/gov/pagopa/pu/registry/repository/InstallmentRegistryRepository.java

```diff
@@ -0,0 +1,9 @@
+package it.gov.pagopa.pu.registry.repository;
+
+import it.gov.pagopa.pu.registry.model.InstallmentRegistry;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+@RepositoryRestResource(collectionResourceRel = "installmentRegistries", path = "installment-registry")
```

Comment:
the path should be in the plural form
```suggestion
@RepositoryRestResource(collectionResourceRel = "installmentRegistries", path = "installment-registries")
```

Reply:
I've picked "registry" because in this case it's a "registry" of installment events and not multiple registries of installment events - made the change